.eslintignore

```diff
@@ -4,6 +4,7 @@ dist
 commitlint.config.js
 prettier.config.js
 jest.config.js
+stitches.config.ts
```

Comment:
We probably want this file to be parsed by ESLint -- if there are annoying rules (like object key sort order) that don't make sense for something like a theme object, feel free to disable those rules on a per-line or per-file basis

Reply:
Agreed - this should be all fixed with my most recent push!

I was running into this [issue earlier](https://stackoverflow.com/questions/63118405/how-to-fix-eslintrc-the-file-does-not-match-your-project-config) - but now it seems fine?! I have changed my node version today (to do with another issue), so I think it must have been related to that. Either way, it should be all good now.